var/spack/repos/spack_repo/builtin/packages/mgard/package.py

```diff
@@ -4,11 +4,12 @@
 
 from spack_repo.builtin.build_systems.cmake import CMakePackage
```

Comment:
```suggestion
from spack_repo.builtin.build_systems.cmake import CMakePackage, CMakeBuilder
```
I'm getting this error when I tested this pr: `Error: NameError: name 'CMakeBuilder' is not defined`

Reply:
Yeah the auto-convert missed it, I ran into this as well in my rebase.

Reply:
I replicated this issue as well. Things should work now. I was able to install in a clean Docker container that compiles the hipcub package.